prow/config/config.go

```diff
@@ -153,6 +153,10 @@ type ProwConfig struct {
 	// DefaultJobTimeout this is default deadline for prow jobs. This value is used when
 	// no timeout is configured at the job level. This value is set to 24 hours.
 	DefaultJobTimeout *metav1.Duration `json:"default_job_timeout,omitempty"`
+
+	// ManagedHmacEntity contains information about all github repositories and organizations which are using
+	// non-global Hmac token.
+	ManagedHmacEntity ManagedHmacEntity `json:"managed_hmac_entity,omitempty"`
```

Comment:
nit: Something like `managed_webhooks` might be a little clearer. GH webhooks are what is really being managed, the HMAC is just a part of that.
Currently there isn't really an indication that this is related to GH webhooks.

Reply:
Done.